addons/website_mail/controllers/main.py

```diff
@@ -87,6 +87,13 @@ def website_message_subscribe(self, id=0, object=None, message_is_follower="on",
         _message_is_follower = message_is_follower == 'on'
         _object = request.registry[object]
 
+        record = _object.browse(cr, uid, _id, context=context).exists()
+        if not record:
+            return False
+
+        _object.check_access_rights(cr, uid, 'read', context)
```

Comment:
It does not fail, but it's wrong:

`    def check_access_rights(self, cr, uid, operation, raise_exception=True): # no context on purpose.`

Reply:
:facepalm: should be fixed now